### ISSUE TITLE
'Show info' does not show info.

### ISSUE THREAD

**Issue by NitroNils** [TeamNewPipe/NewPipe#2366]
'Show info' does not show info but loads the video directly.
From all places where it appears (ie. from Downloads context menu, from What's new, etc.).

This is destructive when one has a limited dataplan.

Android 6
Huawei P8 Lite
```
android:versionCode="730" android:versionName="0.16.1" android:compileSdkVersion="28" android:compileSdkVersionCodename="9" package="org.schabi.newpipe" platformBuildVersionCode="730" platformBuildVersionName="0.16.1">
<uses-sdk android:minSdkVersion="19" android:targetSdkVersion="28"/>
```

**Comment by gkeegan** [comment]
Could not replicate.

**Comment by jimbo1qaz** [comment]
I experienced the same issue.

- Use an external app and tap a Youtube link
- Open it via Newpipe
- (Prefered 'open' action = Always ask)
- I tap "Show info", JUST ONCE.

Result:

- The video player pops up. Pressing Android back button goes to the video info page.

If I tap "Video player" instead of "Show info", pressing the Android back button returns to the previous app, *not* the info page.

Moto G5 Plus, Android 7.0, Newpipe 0.16.2 (F-Droid?).

**Comment by trymeouteh** [comment]
I made a request to remove the Just Once & Always Buttons since it causes confusion and if you press Always by accident then you have to set it back to Always Ask in the settings again

https://github.com/TeamNewPipe/NewPipe/issues/2172

**Comment by jimbo1qaz** [comment]
https://github.com/TeamNewPipe/NewPipe/blob/f98e5cc22d986fe0df33a5e572b0666a249ded8c/app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java#L1159-L1160

- org.schabi.newpipe.MainActivity#handleIntent
  - `boolean autoPlay = intent.getBooleanExtra(VideoDetailFragment.AUTO_PLAY, false);`
- org.schabi.newpipe.util.NavigationHelper#getIntentByLink(android.content.Context, org.schabi.newpipe.extractor.StreamingService, java.lang.String)
  - `getBoolean(context.getString(R.string.autoplay_through_intent_key), false)`
- Nothing else in the code uses `R.string.autoplay_through_intent_key`.

https://github.com/TeamNewPipe/NewPipe/issues/2232 "Re-add ability to autoplay video when directly opened" claims that this feature (reported as a bug in this issue) has been removed.

https://github.com/TeamNewPipe/NewPipe/blob/21a90bb7ee6d46d87750671754b0d05a6807d516/app/src/main/java/org/schabi/newpipe/util/NavigationHelper.java#L497-L499

It was not removed, but the preference entry to turn it on/off was removed from the GUI, while keeping the choice made in an earlier version of the app!

As a result, this invisible config entry causes some older app installations to either exhibit this issue's behavior or not, without any visible configuration differences.

**Comment by NitroNils** [comment]
So, still open?

**Comment by nyanpasu64** [comment]
Clear app data to avoid this issue.

**Comment by Stypox** [comment]
Out of scope after #2907: the autoplay toggle now exists again
